### PR TITLE
Added shop selector to menu footer (#2a7ymj3) 

### LINKED ISSUE #129: Add shop selector to menu footer
## What is the motivation for adding/enhancing this feature?

The shop name and other instance info is shown in the footer of the menu. It feels very natural to click it and change the selected shop.

## What are the acceptance criteria?

Clicking the ship info in the footer should open an inline popover that lets the user select their shop that they would otherwise have to go into the settings page for.
- [ ] ...

Screen Shot 2022-05-06 at 11 24 27 AM

### PR DESCRIPTION
**Pull request by k2maan**
### Related Issues

Closes #129 

### Short Description and Why It's Useful

Implemented changes to resolve infinite loading spinner on refreshJobs.

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

**Comment by adityasharma7** [comment]
@k2maan 
I think we need to handle case for More jobs

**Comment by k2maan** [comment]
@adityasharma7 sir, the more jobs section works as expected with the shop selector footer.

File: src/views/Orders.vue [hunk 1]
```diff
@@ -455,6 +455,23 @@ export default defineComponent({
         });
       return jobAlert.present();
     },
+    fetchJobs(){
+      this.store.dispatch("job/fetchJobs", {
+        "inputFields": {
+          "enumTypeId": "ORDER_SYS_JOB"
+        }
+      });
+      this.store.dispatch("job/fetchJobs", {
```
Comment (adityasharma7) [inline src/views/Orders.vue:464]: I think batch jobs should have enumTypeId as ORDER_SYS_JOB and we are already fetching those jobs

File: src/views/Miscellaneous.vue [hunk 2]
```diff
@@ -145,7 +147,6 @@ export default defineComponent({
     async refreshJobs(event: any) {
```
Comment (adityasharma7) [inline src/views/Miscellaneous.vue:147]: `'event' is defined but never used`

File: src/views/Pipeline.vue [hunk 3]
```diff
@@ -457,21 +457,18 @@ export default defineComponent({
         event.target.complete();
       })
     },
-    async refreshJobs(event: any) {
-      this.isRetrying = true;
+    async refreshJobs(event: any, isRetrying = false ) {
+      this.isRetrying = isRetrying;
       if(this.segmentSelected === 'pending') {
         this.getPendingJobs().then(() => {
-          if(event) event.target.complete();
```
Comment (adityasharma7) [inline src/views/Pipeline.vue:464]: Check if it is not breaking pull to refresh (ion-refresher) 
https://ionicframework.com/docs/api/refresher#complete

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Pipeline.vue (hunk 3 above)
Comment (shashwatbangar) [reply src/views/Pipeline.vue:464]: I tried "pull to refresh" on Pipeline page & loader keeps on spinning.

File: src/views/Orders.vue (hunk 1 above)
Comment (k2maan) [reply src/views/Orders.vue:464]: Sir, the More Jobs section is not visible if we remove the code.

File: src/views/Miscellaneous.vue (hunk 2 above)
Comment (k2maan) [reply src/views/Miscellaneous.vue:147]: Updated.

File: src/views/Pipeline.vue (hunk 3 above)
Comment (adityasharma7) [reply src/views/Pipeline.vue:464]: We could pass event as undefined for all the other cases then pull to refresh

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Pipeline.vue (hunk 3 above)
Comment (k2maan) [reply src/views/Pipeline.vue:464]: Done sir.

**Review by adityasharma7 — APPROVED** [review]

File: src/views/Miscellaneous.vue [hunk 4]
```diff
@@ -14,7 +14,7 @@
           <div v-if="miscellaneousJobs?.length === 0">
             <p class="ion-text-center">{{ $t("There are no miscellaneous jobs right now")}}</p>
             <div class="ion-text-center">
-              <ion-button fill="outline" @click="refreshJobs()">
+              <ion-button fill="outline" @click="refreshJobs(undefined)">
```
Comment (adityasharma7) [inline src/views/Miscellaneous.vue:17]:
```suggestion
              <ion-button fill="outline" @click="refreshJobs()">
```

**Pull request merged**